portfolio.css

```diff
@@ -0,0 +1,174 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  background: linear-gradient(
+    135deg,
+    #feffb6 30%,
+    #cfcece 0,
+    #cfcece 50%,
+    #feffb6 0,
+    #fff 0%,
+    #fff 0
+  );
+  background-size: 1550px 1550px;
+  background-repeat: no-repeat;
+}
+
+.container-portfolio {
+  width: 1440px;
+  margin: 0 auto;
+  padding: 100px 120px;
+}
+
+h1 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 48px;
+  line-height: normal;
+  color: #111111;
+}
+
+.slider__element__number {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 57px;
+  color: #111111;
+  margin-bottom: 32px;
+}
+
+.sterssed {
+  border-bottom: 2px solid #db4951;
+  padding-bottom: 10px;
+}
+
+.slider__element__name {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 29px;
+  color: #111111;
+  margin-bottom: 20px;
+}
+
+.my-works {
+  /* margin: 0 auto; */
+  /* padding: 100px 0 0 120px; */
+}
+
+.slider {
+  position: relative;
+  margin: 0px auto 100px;
+  width: 1200px;
+  height: 610px;
+}
+
+.slider__crop {
+  overflow: hidden;
+}
+
+.slider__container {
+  display: inline-flex;
+  justify-content: flex-start;
+  transition: margin-left 250ms;
+}
+
+.slider__element {
+  position: relative;
+  min-width: 346px;
+  height: 609px;
+  background-color: #fff;
+  padding: 30px 30px 50px 30px;
+  border-radius: 3px;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.15);
+  margin-right: 80px;
+  cursor: pointer;
+}
+
+.slider__element:hover::after {
+  position: absolute;
+  height: 609px;
+  top: 0;
+  left: 0;
+  content: '';
+  border-left: 10px solid #db4951;
+  border-radius: 3px 0px 0px 3px;
+}
+
+.slider__element__text {
+  margin-top: 30px;
+}
+
+.slider__left {
+  position: absolute;
```

Comment:
a lot of similar properties as for slider__right
combine rules

.slider__left, .slider__right